### DISCUSSION TITLE
Disable error reporter logs for test environment

### DISCUSSION THREAD

**Discussion by LeCoupa** [adonisjs/core#2459] [Help]
Is there a way to avoid displaying the error reporter logs when we send a wrong query with Japa? It clutters the results and makes things difficult to read in the terminal.

<img width="977" alt="Screenshot 2021-04-07 at 14 45 59" src="https://user-images.githubusercontent.com/1658644/113868652-35591180-97b0-11eb-8c63-049f5ad7c4d4.png">

```javascript
  test('EmailsController.checkEmail.wrongEmail', async (assert) => {
    const wrongEmail = 'julien.le.coupanec'
    const { body } = await supertest(baseUrl).get(`/check?email=${wrongEmail}`).expect(400)

    assert.equal(body.message, 'E_VALIDATION_FAILURE: Validation Exception')
  })
```

**Comment by thetutlage** [comment]
You can disable it self implementing the report method inside your exception handler, or use the `ignoreCode` property. More info here https://preview.adonisjs.com/guides/http/exception-handling#error-reporting

**Reply by LeCoupa** [reply]
I still see the error when I run `npm run test`. If it can help. All my adonis packages are also up-to-date.

```typescript
export default class ExceptionHandler extends HttpExceptionHandler {
  protected ignoreCodes = ['E_UNAUTHORIZED_ACCESS']
  protected ignoreStatuses = [401]

  constructor() {
    super(Logger)
  }

  public async report(error: Error, context: HttpContextContract) {
    if (Env.get('NODE_ENV') === 'production') {
      Sentry.captureException(error)
    } else {
      console.error(error)
    }

    return super.handle(error, context)
  }
}
```

**Reply by thetutlage** [reply]
There are couple of issues with you code. 

- 1st, you are calling `super.handle` inside the `report` method
- 2nd, you are always calling `super.handle`. Your if/else doesn't stop that line from getting executed
